js/binance.js

```diff
@@ -5255,6 +5257,38 @@ module.exports = class binance extends Exchange {
         return this.filterByArray (result, 'symbol', symbols, false);
     }
 
+    async fetchPositionMode (symbol = undefined, params = {}) {
+        const request = {};
+        let method = undefined;
+        let defaultType = 'future';
+        defaultType = this.safeString (this.options, 'defaultType', defaultType);
+        const type = this.safeString (params, 'type', defaultType);
```

Comment:
Let's use `handleMarketTypeAndParams` here (see how it's done in the other methods in this file).